From c860dcbe63b0e393c95bfb0131238f91aaac11d3 Mon Sep 17 00:00:00 2001
From: Alexander Kanavin <alex@linutronix.de>
Date: Thu, 7 Oct 2021 12:44:18 +0200
Subject: [PATCH] build: do not use autoconf's config.sub to 'canonicalize'
 names

The outcome is that processed names no longer match our custom rust
target definitions, and the build fails.

Upstream-Status: Inappropriate [oe-core specific]
Signed-off-by: Alexander Kanavin <alex@linutronix.de>

---
 build/moz.configure/init.configure | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/build/moz.configure/init.configure b/build/moz.configure/init.configure
index 81f500a0b7..0b7a2ff60f 100644
--- a/build/moz.configure/init.configure
+++ b/build/moz.configure/init.configure
@@ -585,7 +585,7 @@ def help_host_target(help, host, target):
 
 def config_sub(shell, triplet):
     config_sub = os.path.join(os.path.dirname(__file__), "..", "autoconf", "config.sub")
-    return check_cmd_output(shell, config_sub, triplet).strip()
+    return triplet
 
 
 @depends("--host", shell)
